contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -52,13 +56,19 @@ func constructor{
     ERC20_initializer(name, symbol, decimals)
     ERC20_mint(recipient, initial_supply)
     Ownable_initializer(owner)
+    # TODO Init reward token from controller
     return ()
 end
 
 #
 # Getters
 #
 
+@view
+func get_REWARD_TOKEN{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (token: felt):
+    return REWARD_TOKEN.read()
```

Comment:
does this one-liner works? usually in cairo, one needs to separate reading from returning ie

```
let (res) = REWARD_TOKEN.read()
return res
```

Reply:
These are fine, starkware added this feature a little while ago. The return of a function is a tuple-like thing (being vague here because they are too, tuples are not a native type but they're technically the return type of functions.. 🤷) So you can just return a function call with the same return type.

Reply:
Thanks, I wasn't aware of that - and indeed I should have returned (token) in the snippet above.